builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -284,6 +284,10 @@ def loadServiceConfigurationData() {
 
     if (service_name.trim() == "jazz_admin") {
       updateConfigValue("{jazz_admin}", config_loader.JAZZ.ADMIN)
+      updateConfigValue("{conf-region}", region)
+      sh "sed -i -- 's/{config_table}/${config_loader.INSTANCE_PREFIX}_JazzConfig/g' ./config/dev-config.json"
```

Comment:
use the same method updateConfigValue here for replacing {config_table}

Reply:
done